src/services/user.service.ts

```diff
@@ -40,9 +40,9 @@ export const userService = {
     credentialsExist() {
         // Don't try to log in if we don't have any credentials
         if (getAuthToken() || getRefreshToken()) {
-            return of(true);
+            return true;
```

Comment:
Put this as it was, as it is not meant to be used outside rxjs for now. I can see the reasoning here, but you started using this service in the comp, and is not meant to be used there.

Reply:
fixed